### DISCUSSION TITLE
Getting Exif Data

### DISCUSSION THREAD

**Discussion by SoneThaya** [ionic-team/capacitor#5521] [Q&A] [answered]
I am trying to get exif data from an image either by taking a picture or from the gallery. There is an example code on how to take photo on the ionic camera website but no example for getting exif data. I am new to Ionic Angular and looking for some help on how to implement getting exif data. Thanks.

This function returns an array with `webPath` and `format` but not `exif`.

```
async takePicture() {
    const image = await Camera.pickImages({
      quality: 90,
      width: 200,
      height:200,
      limit: 1
    });
    console.log('imageUrl', image);
  }
```

**Comment by jcesarmobile** [comment, marked as answer]
I guess you are running on web, it’s not implemented there, only for iOS and Android.

**Reply by SoneThaya** [reply]
Thank you.
